src/js/components/AppVolumesListComponent.jsx

```diff
@@ -40,10 +40,13 @@ var AppVolumesListComponent = React.createClass({
 
   getVolumeType: function (volume) {
     if (volume.hostPath != null) {
-      return LocalVolumesConstants.TYPES.DOCKER;
+      return VolumesConstants.TYPES.DOCKER;
     }
     if (volume.persistent != null || volume["size"] != null) {
-      return LocalVolumesConstants.TYPES.LOCAL;
+      return VolumesConstants.TYPES.LOCAL;
+    }
+    if (volume.external != null) {
+      return VolumesConstants.TYPES.NETWORK;
```

Comment:
I wonder if we should rename these 2 to at match the API nomenclature : `TYPES.PERSISTENT` and `TYPES.EXTERNAL`.